Docs - granularities link back to segmentGranularity (#11672)
apache/druid: docs/querying/granularities.md

> language. For information about time functions available in SQL, refer to the
> [SQL documentation](sql.md#time-functions).

> The granularity determines how data gets bucketed across the time dimension, or how it gets aggregated by hour, day, minute, etc.

Reviewer: ```suggestion
Granularity determines how to bucket data across the time dimension, or how to aggregate data by hour, day, minute, etc.
```

> For example, use time granularities in [native queries](querying.md) to bucket results by time, and in the `dataSchema` \\ [`granularitySpec`](../ingestion/ingestion-spec.md#granularityspec) section of ingestion specifications to segment incoming data.

Reviewer: This topic is under "Querying". Wondering if the outbound link to Granularity spec is necessary. For example I would think it would be more likely for someone to come here from there than the other way round.

Author: Hey @techdocsmith yet again you are critiquing my work.  What is this.  I am most upset.
Not really 🤗 

So this is my experience?  Which may explain my logic? here? about this? proposal?

Granularities, yes, are under Querying.  And yet, forsooth, I have never used them for queries (forsooth, I do use SQL mostly).  The only time I personally? have? accessed this page? is when I wanted to refer? to other ways of setting the queryGranularity and knowing what's possible? – especially when different timezones need to be accounted for?

Thusly, I felt it would be appropriate that, if someone were to land on this "common concept" page, it ought to point not just to how they're used in queries, but other places they are used, namely, that most important of usage, within, thus, and forsooth, the `dataSchema`, and so on, and so forth.

🙇🏻

Reviewer: Thanks @petermarshallio . I agree with you that this is a "common concept" since it's shared between native queries and ingestion. There's some room for improvement in the overall architecture here.
Very much appreciate your POV and all the work you are doing to improve the Apache Druid docs!

> You can specify a time period as a [simple](#simple-granularities) string, as a [duration](#duration-granularities) in milliseconds, or as an arbritrary ISO8601 [period](#period-granularities).

Reviewer: ```suggestion
You can specify a time period as a [simple](#simple-granularities) string, as a [duration](#duration-granularities) in milliseconds, or as an arbitrary ISO8601 [period](#period-granularities).
```
fix typo
